contracts/bonding/BondingManager.sol

```diff
@@ -587,6 +587,8 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
             delegationAmount = delegationAmount.add(currentBondedAmount);
 
             decreaseTotalStake(currentDelegate, currentBondedAmount, _oldDelegateNewPosPrev, _oldDelegateNewPosNext);
+            // currentDelegate is always different than msg.sender so no need to avoid double checkpointing
```

Comment:
Suggest changing msg.sender to _owner in this comment.

And just to clarify - this statement is only true because L582 prevents execution from reaching here if the address is a transcoder (i.e. self-delegated) correct?

Reply:
> And just to clarify - this statement is only true because L582 prevents execution from reaching here if the address is a transcoder (i.e. self-delegated) correct?

Correct! I've rephrased the doc, including the `_owner` fix you suggested